requirements.txt

```diff
@@ -16,6 +16,8 @@ Flask-Babel
 livereload
 python-dateutil
 
+pathlib
```

Comment:
We are only supporting python 3 so this is not really necessary..

Reply:
But won´t really break anything either, will it?

Reply:
No it's fine